hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconConstants.java

```diff
@@ -58,4 +58,9 @@ private ReconConstants() {
       + PIPELINE_DB_SUFFIX;
   public static final String RECON_SCM_NODE_DB =
       "recon-node.db";
+  public static final long MAX_FILE_SIZE_UPPER_BOUND = 1125899906842624L;
```

Comment:
```suggestion
  // 1125899906842624L = 1PB
  public static final long MAX_FILE_SIZE_UPPER_BOUND = 1125899906842624L;
```

move the original comment here as well